### PR TITLE
[ui] Use /api/v1/info endpoint to discover ipv4/ipv6 api addresses

### PR DESCRIPTION
**Pull request by chantra**

### CODE REVIEW

File: src/react/jsx/atc.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var ShapingPanel = require('./shaping');
 var Atc = React.createClass({
   getInitialState: function() {
     return {
-      client: new AtcRestClient(this.props.primary, this.props.secondary, this.props.endpoint),
+      client: new AtcRestClient(),
```
Comment (zfjagann) [inline src/react/jsx/atc.js]: You sure you don't want:

    client: new AtcRestClient(this.props.endpoint),

**Comment by zfjagann** [comment]
We still have files in `/static/js`? I thought we got rid of them in #196 ?

File: src/react/jsx/atc.js (hunk 1 above)
Comment (chantra) [inline src/react/jsx/atc.js]: yeah, for the sake of this diff, I was assuming that the default was fine.... mainly if we want to separate the ui from the api.... but yes, there may be situation where it does not work.

Comment (zfjagann) [inline src/react/jsx/atc.js]: Ok, just checking to be sure it wasn't a typo.

**Pull request merged**